app/domain/output_document.rb

```diff
@@ -63,7 +63,7 @@ def lead
   end
 
   def appointment_reference
```

Comment:
Perhaps this should be called `output_document_reference`? As it'll (eventually) be related to the output document issue, and not necessarily the appointment reference - right?